### ISSUE TITLE
Original file contains multiple headwords for the same card

### ISSUE THREAD

**Issue by danchenkov** [Tvangeste/dsl2mobi#3]
I get this Error processing Sample.dsl (and any other DSL file)

```
ERROR: Original file contains multiple headwords for the same card: #NAME      "Sample for Testing (En-Ru)"
```

Gentoo, Ruby 1.9.3, UTF-8

**Comment by Tvangeste** [comment]
Just fixed in 4a1476a.

**Comment by Alekssavin** [comment]
I have the same problem. 
First I tried converting my dsl dictionary, I got: 
`ERROR: Currently not supported DSL encoding: UTF-16LE
INFO: Convert the DSL file into UTF-8 before running this script.`
I converted the initial file with iconv tool, run your script again, got the error from the name of the topic:
`ERROR: Original file contains multiple headwords for the same card: #NAME      "Sample for Testing (En-Ru)"`
